Makefile

```diff
@@ -1,23 +1,16 @@
-UGLIFY = node_modules/.bin/uglifyjs
```

Comment:
there is still release script left there, personally im not really fond of Makefiles, but i assumed its preferred by the Ramda's core team, so I have left it in place